### ISSUE TITLE
Feature request: support for wikis in subdirectories

### ISSUE THREAD

**Issue by ricardoboss** [gollum/gollum#2004]
Hi, great project!

I wanted to be able to preview the wiki I am working on (while researching if my request was already open I found https://github.com/gollum/gollum/issues/1031, which is what I probably would have stumbled upon next).

To do that, I wanted to run gollum in a docker container, which worked fine.
The part that not worked is the actual wiki part, since it is a directory in my repository and not its own repository.

In short: I want to be able to tell gollum to use a subdirectory as the wiki root (instead of the repository root) or use gollum on a directory without a git repository.

Is this possible?

**Comment by dometto** [comment]
Hi @ricardoboss! You can use the `--page-file-dir` command line option (see the readme) to specify a subdirectory of the repository to use for the wiki. It has to be a committed directory though, no support for uncommitted files exists at the moment.
